app/resources/lang/en/components.php

```diff
@@ -45,5 +45,6 @@
         'current' => 'Parliament is meeting in :location this week.',
         'last' => 'The last plenary session was held <strong>from :start to :end</strong> in :location.',
         'next' => 'The next plenary session will be held in :location <strong>from :start to :end</strong>.',
+        'agenda' => 'Take a look at this session\'s votes in its <a href=":agenda" target="_blank" rel="noopener noreferrer">agenda</a>.',
```

Comment:
I have a suggestion for slightly easier wording:

> Take a look at [the agenda]() for this session.

Or something like this, although we would have to include two variants for upcoming/past sessions:

> Take a look at [the agenda]() to find out what MEPs have voted on.
> Take a look at [the agenda]() to find out what MEPs will be voting on.